app/src/app/components/stix/timestamp-property/timestamp-view/timestamp-view.component.html

```diff
@@ -1,3 +1,6 @@
 <div class="app-timestamp-view">
+    <span *ngIf="config.displayCreatorUsernameWithTimestamp && username !== ''">
+      <b>{{username}}</b>, 
```

Comment:
This should show the user's [displayName](https://github.com/center-for-threat-informed-defense/attack-workbench-frontend/blob/develop/app/src/app/classes/authn/user-account.ts#L14) rather than their `username`